libvmi/libvmi.h

```diff
@@ -55,6 +56,8 @@ extern "C" {
 
 #define VMI_INIT_DOMAINUUID (1u << 3) /**< initialize using domain uuid */
 
+#define VMI_INIT_WAIT_FOR_DOMAIN (1u << 4) /**< initialize using a domain watcher */
```

Comment:
Not sure this is a good name if we are looking for domain deletion events

Reply:
Can you suggest a better name to go on?

Reply:
How about `VMI_INIT_DOMAINWATCH`?

Reply:
Sure looks in check with all the other names